content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -2,94 +2,98 @@
+create components beforehand by going to the :menuselection:`Top Menu --> Products --> Create`, and
+add them to the BoM later on. 
 
 
 .. image:: bill_configuration/bom_1.png
     :align: center
+    :alt: Set up a Bill of Materials.
 
 .. warning::
    The destination location should **not** be a scrap location. A scrap location is where you put
    products that you don't need.
 
-Using the same BoM to describe Variants
+Using BoMs with Product Variants
 ---------------------------------------
 
-As suggested above, you can use *BOMs* for specific *Product Variants*. 
-Once the various attributes have been configured on the product form, there 
-are two ways to configure the appropriate BoM for the respective product combination. 
+As suggested above, you can assign BoMs to specific *Product Variants*. There are two ways to set
+up BoMs for product variants, after the attributes have been configured on the product form.
 
-Either create one BoM per variant, by specifying the Product Variant
-in the dedicated field below the product name. Or use one BOM, that contains
-all of the components and for each component, indicate which variant it applies to, 
-using the *Apply on Variants* column, as shown below. 
+You can create one BoM per variant by specifying the *Product Variant* in the dedicated field below
+the product name. Alternatively, you can use one master BoM that contains all of the components, and
+specify which variant each component applies to using the *Apply on Variants* column, as shown
+below. 
 
 .. image:: bill_configuration/bom_2.png
     :align: center
+    :alt: Product Variants in the Bill of Materials. 
 
 
 Adding Operations
 =================
 
-You can also add operations to your *BoM*, if you want workers to follow instructions
-or register time spent. To use this feature, enable the *Work Orders* feature in the 
-*Manufacturing* app settings, as shown below.
+You can also add *Operations* to your BoM if you want workers to follow instructions or register
+time spent. To use this feature, enable the *Work Orders* feature in the *Manufacturing* app
+settings, as shown below.
 
 .. image:: bill_configuration/bom_3.png
     :align: center
+    :alt: Work Orders Setting. 
 
 .. note::
-         Each operation is unique as it is always linked to only one BOM. This being said, 
-         Operations Operations can be re-used when configuring a new BOM, with the *Copy Existing 
+         Each operation is unique, as it is always exclusively linked to one BoM. 
+         Operations can be re-used when configuring a new BoM, with the *Copy Existing 
          Operations* feature.
 
 .. image:: bill_configuration/bom_4.png
     :align: center
+    :alt: *Copy Existing Operations* feature. 
 
-Finally, just like for components, operations can also be variant specific only, as shown below.
+Like components, operations can also be variant-specific, as shown below.
 
 .. image:: bill_configuration/bom_5.png
```

Comment:
This image is not really necessary. I would delete the image block and its tags from the code and remove the image file with the command terminal using `git rm [file-path]`